src/main/java/com/google/api/gax/core/PropertiesProvider.java

```diff
@@ -68,22 +69,30 @@ public static String loadProperty(Class<?> loadedClass, String propertyFilePath,
       Properties properties = new Properties();
       properties.load(loadedClass.getResourceAsStream(propertyFilePath));
       return properties.getProperty(key);
+    } catch (NullPointerException e) {
+      fileNotFoundWarning(loadedClass, propertyFilePath);
     } catch (Exception e) {
-      e.printStackTrace(System.err);
+      System.err.println(e.getStackTrace());
     }
     return null;
   }
 
   private static String loadGaxProperty(String key) {
     try {
       if (gaxProperties.isEmpty()) {
-        gaxProperties.load(
-            PropertiesProvider.class.getResourceAsStream("/com/google/api/gax/gax.properties"));
+        gaxProperties.load(PropertiesProvider.class.getResourceAsStream(GAX_PROPERTY_FILE));
       }
       return gaxProperties.getProperty(key);
+    } catch (NullPointerException e) {
+      fileNotFoundWarning(PropertiesProvider.class, GAX_PROPERTY_FILE);
     } catch (Exception e) {
-      // Ignore
+      System.err.println(e.getStackTrace());
     }
     return null;
   }
+
+  private static void fileNotFoundWarning(Class<?> loadedClass, String propertyFilePath) {
```

Comment:
add "print" at the beginning of the function name; also add quotes around the `%s`